app/src/main/java/com/poupa/vinylmusicplayer/dialogs/SongDetailDialog.java

```diff
@@ -146,6 +146,14 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
                 ? String.format(Locale.getDefault(), "%s: %.2f dB ", context.getString(R.string.album), song.replayGainAlbum)
                 : "- ";
         htmlBuilder.appendLine(R.string.label_replay_gain, rgTrack, rgAlbum);
+
+        final String rgPeakTrack = song.replayGainPeakTrack != 1.0f
+                ? String.format(Locale.getDefault(), "%s: %.2f ", context.getString(R.string.song), song.replayGainPeakTrack)
```

Comment:
What is the unit of RG peak please? If it's dB, can the unit be added in the output string?

Reply:
I think it's a relative amplitude (not in dB), so it does not have a unit